hudi-utilities/src/main/java/org/apache/hudi/utilities/HDFSParquetImporter.java

```diff
@@ -64,20 +65,17 @@
 
   private static volatile Logger log = LogManager.getLogger(HDFSParquetImporter.class);
 
-  public static final SimpleDateFormat PARTITION_FORMATTER = new SimpleDateFormat("yyyy/MM/dd");
-  private static volatile Logger logger = LogManager.getLogger(HDFSParquetImporter.class);
+  private static final DateTimeFormatter PARTITION_FORMATTER = DateTimeFormatter.ofPattern("yyyy/MM/dd")
+      .withZone(ZoneOffset.UTC);
```

Comment:
can we leave this to the default zone on the box? typically, servers run in UTC anyway